.github/workflows/CI.yml

```diff
@@ -30,6 +30,8 @@ jobs:
       - name: Run the testsuite
         shell: bash
         run: |
+         # temporary workaround
```

Comment:
Can you explain why we need that workaround, so that we can remove it in the future ;-)